snapshot-labs/sx-monorepo
# feat: boost tab with boosts count

### Summary

Closes: https://github.com/snapshot-labs/workflow/issues/323

- Add a new tab boost with count of boosts
![image](https://github.com/user-attachments/assets/db973ffd-0fe5-45ab-a704-6fe8ba1a542c)


### How to test

1. Go to http://127.0.0.1:8080/#/s:acrossprotocol.eth/proposal/0x078c419b439cf5e2889c3e2a14ac2953d8b3b033cf313254b61be4fbbc333483
2. You should see a new tab "Boost" with the count 
3. On click, it should redirect to v1
